core/notifications/notifications.go

```diff
@@ -132,6 +241,55 @@ func (n *notificationService) List(limit int64, includeRead bool) ([]*model.Noti
 	return result, nil
 }
 
+func (n *notificationService) GetLastNotificationId(acl string) string {
+	n.RLock()
+	defer n.RUnlock()
+	return n.lastNotificationIdToAcl[acl]
+}
+
 func (n *notificationService) isNotificationRead(notification *model.Notification) bool {
 	return notification.GetStatus() == model.Notification_Read || notification.GetStatus() == model.Notification_Replied
 }
+
+func (n *notificationService) loadNotificationObject(ctx context.Context) {
+	uk, err := domain.NewUniqueKey(sb.SmartBlockTypeNotificationObject, "")
+	if err != nil {
+		log.Errorf("failed to get notification object unique key: %v", err)
+		return
+	}
+	techSpaceID, err := n.spaceCore.DeriveID(ctx, spacecore.TechSpaceType)
```

Comment:
I guess we shouldn't get tech space like this, maybe it is better to expose a method that returns a tech space from the space service (if it is not exposed yet).